### ISSUE TITLE
Explosions should create damaged tiles

### ISSUE THREAD

**Issue by Flareguy** [space-wizards/space-station-14#33423]
## Description
![Hippie Station 13 Widdershins Fishery XVI 2014-11-28 211013](https://github.com/user-attachments/assets/a18c7210-4182-46b6-8b45-ebc14898e1fb)

Explosions should "damage" tiles, like in SS13. This is generally a massive boon for immersion and also generally looks amazing. It's a crying shame we somehow don't have these yet.
This has a good amount of things that need to be considered, though, so they will be given their own section.

## Implementation: Sprites
These implementation segments are not meant to be highly technical. They simply exists to discuss some incredibly important caveats about how these are implemented.

Starting with the single most important thing about this: **how the sprites are handled.** A generic overlay used for all tiles is not sufficient, as explosion graphics are made with the details of the tiles in mind. For example: the only currently-existing ones in SS14, damaged steel tiles, draw their damage out from the center.

In other words, *tiles will need a way to define what they look like when damaged by an explosion.* There are many different possible ways you could go with this that I will discuss in a separate issue, because I do not want to overload this one with  dubiously useful information.

## Implementation: Damage Radius
As you can see in this random image from the /tg/ fourms: the higher the explosion power, the more damaged the tile will become. Basically, the closer to the epicenter of the explosion you are, the more plating will show up.
Not shown: this also means that weak bombs will also usually not create any plating, and just damage tiles.

![a6f8PCy](https://github.com/user-attachments/assets/398b5964-fbbe-454e-a91f-204a1670dcd6)

**Comment by Flareguy** [comment]
i sent this way too early. oops. i will reopen this once i am actually finished typing
